modules/custom-api.xql

```diff
@@ -53,6 +54,8 @@ declare function api:persons-all-list($request as map(*)) {
                 })]
             )
 
+    let $items := $searchitems[fn:sum(ft:field(., 'sent-count', 'xs:integer'), ft:field(., 'received-count', 'xs:integer')) > 0]
```

Comment:
To allow switching between correspondents and mentioned persons, see example at: https://www.briefedition.alfred-escher.ch/kontexte/personen/?category=A&search=&view=correspondents